### PR TITLE
Added Register and Login files

### PR DESCRIPTION
**Pull request by nadinefrawley**

### CODE REVIEW

File: system/models/users/register.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import os
+
+class Register:
+    def __init__(self, user_id: int, user_name: str, password: str, is_admin: bool, error: str):
+        self._user_id = user_id
+        self._user_name = user_name
+        self._password = password
+        self._is_admin = is_admin
+
+    def ask_user_credentials(self):
```
Comment (dylancoffey99) [inline system/models/users/register.py:11]: Make sure to type cast the return value with "def ask_user_credentials(self): -> return value type"
Although this implementation looks like its only temporary so thats fine for now.

File: system/models/users/login.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class Login:
+    def __init__(self, user_id: int, user_name: str, password: str, is_admin: bool, error: str):
```
Comment (dylancoffey99) [inline system/models/users/login.py:2]: I'm not 100% sure on how this implementation will work but the login/register should pass in a User object, initialise it as well as a UserDB (hasn't been merged yet, waiting for a review). These are then used for the various authentication methods.

So I think something like this:
def __init__(self, user: User):
    self._user = user
    self._user_db = UserDB("userDB")

**Review by dylancoffey99 — CHANGES_REQUESTED** [review]
I think these files should also be put in the authentication package instead.

**Comment by Miracle2001** [comment]
I agree with Dylan.

**Comment by dylancoffey99** [comment]
I'm going to close this pull request as its too outdated and doesn't incorporate the MVC pattern. I think it's better to merge my latest pull request to the master, and start over again:

https://github.com/dylancoffey99/CS4125-Wholesaler-System-Iridium/pull/12

**Pull request closed without merging**